config/application.rb

```diff
@@ -20,17 +17,23 @@ class Application < Rails::Application
       g.stylesheets     false
       g.javascripts     false
     end
-    # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
 
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration can go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
     config.i18n.load_path += Dir[Rails.root.join("config", "locales", "**",
         "*.{rb,yml}")]
     config.i18n.available_locales = [:en, :vi]
     config.i18n.default_locale = :vi
     config.action_view.embed_authenticity_token_in_remote_forms = true
+
+    config.generators do |g|
+      g.test_framework :rspec,
+        :fixtures => true,
```

Comment:
đổi hết cú pháp cũ thành cú pháp mới nak :v
```suggestion
        fixtures: true,
```

Reply:
vâng anh.